src/cdk/v2/destinations/webhook/procWorkflow.yaml

```diff
@@ -0,0 +1,94 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+    exportAll: true
+  - path: ../../../../v0/destinations/webhook/utils
+  - name: getHashFromArray
+    path: ../../../../v0/util
+  - name: getIntegrationsObj
+    path: ../../../../v0/util
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+
+steps:
+  - name: checkIfProcessed
+    condition: .message.statusCode
+    template: |
+      $.batchMode ? .message.body.JSON : .message
+    onComplete: return
+  - name: messageType
```

Comment:
Is this step required? looks it is only used in one validateInput step

Reply:
addressed